src/fiatExchanges/quotes/normalizeQuotes.ts

```diff
@@ -34,8 +35,14 @@ export const quotesByFeeComparator = (quote1: NormalizedQuote, quote2: Normalize
     [Currency.Dollar]: '1',
     [Currency.Euro]: '1',
   }
-  const providerFee1 = quote1.getFeeInFiat(exchangeRates) ?? new BigNumber(Infinity)
-  const providerFee2 = quote2.getFeeInFiat(exchangeRates) ?? new BigNumber(Infinity)
+  // also dummy token info. all we need is the usdPrice
+  const dummyTokenInfo = {
```

Comment:
I commented on this already in NormalizedQuote, but this is a good example of where the type is misleading. If the second parameter was actually optional, you shouldn't have to do something like this.